src/main/java/blackjack/view/InputView.java

```diff
@@ -0,0 +1,52 @@
+package blackjack.view;
+
+import java.util.List;
+import java.util.Scanner;
+
+import blackjack.domain.gamer.Player;
+
+public class InputView {
+	private static final InputView INSTANCE = new InputView(new Scanner(System.in));
+
+	private static final String PLAYER_NAME_DELIMITER = ",";
+	private static final String COMMAND_YES = "y";
+	private static final String COMMAND_NO = "n";
+	private final Scanner scanner;
+
+	private InputView(Scanner scanner) {
+		this.scanner = scanner;
+	}
+
+	public static InputView getInstance() {
+		return INSTANCE;
+	}
+
+	public List<String> readPlayerNames() {
+		System.out.println("게임에 참여할 사람의 이름을 입력하세요.(쉼표 기준으로 분리)");
+		String names = scanner.nextLine().replace(" ", "");
+
+		validatePlayerNamesFormat(names);
+		return List.of(names.split(PLAYER_NAME_DELIMITER));
+	}
+
+	private void validatePlayerNamesFormat(String names) {
+		if (names.startsWith(PLAYER_NAME_DELIMITER) || names.endsWith(PLAYER_NAME_DELIMITER)) {
+			throw new IllegalArgumentException("플레이어의 이름은 쉼표(,)로 시작하거나 끝날 수 없습니다.");
+		}
+	}
+
+	public String readHitOrStand(Player player) {
+		System.out.println(
+			String.format("%s는 한장의 카드를 더 받겠습니까?(예는 %s, 아니오는 %s)", player.getName(), COMMAND_YES, COMMAND_NO));
+		String command = scanner.nextLine();
+		validateCommand(command);
+
+		return command;
+	}
+
+	private void validateCommand(String command) {
+		if (!List.of(COMMAND_YES, COMMAND_NO).contains(command)) {
+			throw new IllegalArgumentException(String.format("%s 또는 %s만 입력할 수 있습니다.", COMMAND_YES, COMMAND_NO));
```

Comment:
```suggestion
			throw new IllegalArgumentException(String.format("y 또는 s만 입력할 수 있습니다.");
```
나중에 실제 에러를 디버깅 할때 혼란만 더 줄수 있을것 같아서요

Reply:
y,n 등의 커멘드를 현재 InputView의 두 메서드와 컨트롤러의 한 메서드에서 사용하고 있어서
여러 곳에서 사용하고 있기 때문에 하나의 공간에서 관리하도록 하고 싶습니다.

현재 `COMMAND_` 라는 이름만 붙은 String 으로 관리하고 있기 때문에 혼란을 줄 수 있다는 생각이 드는데
enum으로 관리하면 괜찮지 않을까 싶습니다.

Command라는 enum으로 만들어서 관리하는 것은 별로인가요??
한번 변경해볼테니 리뷰 부탁드립니다!

Reply:
> Command는 도메인 아닌가요? ㅎㅎ view에서 도메인이 있게되네요

뷰에서만 사용하는 값들에 대해 관리하고 있기 때문에 `Command`를 도메인으로 생각하지는 않았습니다!
특히 y,n 이라는 값은 view, controller 에서만 사용되고 있어서 더더욱 뷰 로직이라고 생각이 되었는데요!
혹시 뷰 단에 Enum이 있는 것은 이상할까요??

만약 이상하다면 이유와 어떻게 개선하는 것이 좋을 지 조언 주신다면 감사하겠습니다:) !